python/src/gz/sim/Joint.cc

```diff
@@ -94,7 +94,7 @@ void defineSimJoint(py::object module)
       py::arg("ecm"),
       py::arg("limits"),
       "Set the effort limits on a joint axis.")
-  .def("set_position_imits", &gz::sim::Joint::SetPositionLimits,
+  .def("set_position_limits", &gz::sim::Joint::SetPositionLimits,
```

Comment:
Shouldn't we be deprecating this instead of removing it.

Reply:
Yeah, I think we should deprecate the old function and add the new one.

Reply:
https://github.com/gazebosim/gz-sim/pull/2578/commits/27a46daeeef890f949514b88b086daf3eb8312ea

Reply:
@azeey this is wrong in older versions, what should we do?